### PR TITLE
Adding building, training, redundancy and failover to the boundary list

### PR DESCRIPTION
**Pull request by vaughanknight**
Adding building, training, redundancy and failover to the boundary list, details in Issue #78 

3 bullet points added - Building, Training, Redundancy and failover
Formatting - the paragraph following the boundary list is now separated so it doesn't append to the last bullet point.

### CODE REVIEW

File: Software_Carbon_Intensity/Software_Carbon_Intensity_Specification.md [hunk 1]
```diff
@@ -144,9 +144,13 @@ The calculation of software carbon intensity MUST include all supporting infrast
 - idle machines
 - logging
 - scanning
+- building
 - testing
+- training
 - operations
 - backup
+- redundancy and failover
```
Comment (atg-abhishek) [inline Software_Carbon_Intensity/Software_Carbon_Intensity_Specification.md:152]:
```suggestion
- resources to support redundancy
- resources to support failover
```

**Review by atg-abhishek — CHANGES_REQUESTED** [review]
@vaughanknight looks good, I've made a minor suggestion that I think clarifies redundancy and failover better.

**Review by vaughanknight — COMMENTED** [review]
Wording updates for clarity done in line with your reocmmendations @atg-abhishek

**Review by atg-abhishek — APPROVED** [review]
LGTM!

**Comment by atg-abhishek** [comment]
Approved by WG during meeting

**Pull request merged**